app/domain/logs.rb

```diff
@@ -489,14 +490,69 @@ module AuthnJwt
         code: "CONJ00096D"
       )
 
-      FOUND_JWT_FIELD_IN_TOKEN = ::Util::TrackableLogMessageClass.new(
+      FoundJwtFieldInToken = ::Util::TrackableLogMessageClass.new(
         msg: "Successfully found field '{0}' in token and its value is '{1}'",
         code: "CONJ00097D"
       )
 
-      FOUND_JWT_IDENTITY= ::Util::TrackableLogMessageClass.new(
+      FoundJwtIdentity= ::Util::TrackableLogMessageClass.new(
         msg: "Successfully found jwt identity '{0}'",
-        code: "CONJ00098D"
+        code: "CONJ00098I"
+      )
+
+      ValidatedJwtStatusConfiguration = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated JWT status configuration",
+        code: "CONJ00099I"
+      )
+
+      ValidatedAccountExists = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated that account exists",
+        code: "CONJ00100D"
+      )
+
+      ValidateAndDecodeTokenPassed = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully passed validate and decode token",
+        code: "CONJ00101D"
+      )
+
+      GetJwtIdentityPassed = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully passed get JWT Identity",
+        code: "CONJ00102D"
+      )
+
+      ValidateRestrictionsPassed = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully passed JWT validate restrictions",
+        code: "CONJ00103D"
+      )
+
+      CreateValidateAndDecodeTokenInstance = ::Util::TrackableLogMessageClass.new(
+        msg: "Creating token validator instance",
+        code: "CONJ00104D"
+      )
+
+      CreatedValidateAndDecodeTokenInstance = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully created token validator instance",
+        code: "CONJ00105D"
+      )
+
+      CreateJwtIdentityProviderInstance = ::Util::TrackableLogMessageClass.new(
+        msg: "Creating JWT identity provider instance",
+        code: "CONJ00106D"
+      )
+
+      CreatedJwtIdentityProviderInstance = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully created JWT identity provider instance",
+        code: "CONJ00107D"
+      )
+
+      CreateJwtRestrictionsValidatorInstance = ::Util::TrackableLogMessageClass.new(
+        msg: "Creating JWT restrictions validator instance",
```

Comment:
what is a JWT restrictions validator instance??

Reply:
this means that i am creating the object of the validation i am going to use later

Reply:
Is this what you mean?
```suggestion
        msg: "Creating validator instance for JWT restrictions",
```

Reply:
fixed as we agreed please review